src/app/frontend/common/components/labels/labels.html

```diff
@@ -23,7 +23,9 @@
   <div class="kd-chips kd-chips-switch" ng-show="labelsCtrl.isMoreAvailable()"
        ng-click="labelsCtrl.switchLabelsView()">
     {{labelsCtrl.isShowingAll() ?
-      labelsCtrl.i18n.MSG_LABELS_SHOW_LESS_TOOLTIP : labelsCtrl.i18n.MSG_LABELS_SHOW_ALL_TOOLTIP}}
+      '[[show less labels|Tooltip on the "show less" button for the labels of any kubernetes resource. ' +
```

Comment:
Hmm... I think that `'` are wrong here. You should not add the `' +` part, I think.